apps.kruise.io/CloneSet/health.lua

```diff
@@ -0,0 +1,18 @@
+hs={ status = "Progressing", message = "Waiting for initialization" }
+
+if obj.status ~= nil then
```

Comment:
plz check obj. generation == obj.status.observedGeneration first, otherwise the controller may have not reconcile the workload yet